src/module_PI/Raspberry_PI/main/EmailNotifier.java

```diff
@@ -0,0 +1,239 @@
+package module_PI.Raspberry_PI.main;
+
+import java.io.*;
+import java.util.*;
+
+import javax.activation.*;
+import javax.mail.*;
+import javax.mail.internet.*;
+
+public class EmailNotifier {
+
+	private String gmailUsername;
+	private String gmailPassword;
+
+	private String recipientEmailAddress;
+	private String emailSubject = "Calico Home Security";
+	private String emailMessage;
+	private String attachmentFilePath;
+
+	private Properties mailSessionProperties;
+	private Session javaMailSession;
+	private Message javaMailMessage;
+	private BodyPart messageBodyPart;
+	private Multipart multipart;
+	private DataSource attachmentDataSource;
+
+	public EmailNotifier(String recipientEmailAddress, String message) {
+		this(recipientEmailAddress, message, null);
+	}
+
+	public EmailNotifier(String recipientEmailAddress, String message, String pathToAttachment) {
+		this.loadGmailCredentials();
+		this.setRecipientEmailAddress(recipientEmailAddress);
+		this.emailMessage = message;
+		this.setAttachmentFilePath(pathToAttachment);
+		buildPropertiesAndCreateSession();
+	}
+
+	private void loadGmailCredentials() {
+		try {
+			Properties gmailProperties = new Properties();
+			FileInputStream in = new FileInputStream("gmailCredentials");
```

Comment:
Okay, yeah it does.
In my defense! I am familiar with how java visibility works, but I thought you were saying that the properties file belong to Java or the Java system, making them outside of the scope of regular java code. I now see the error of my ways.
I'll fix it tonight.